1.train_models/nbconverted/classify_genotypes.py

```diff
@@ -54,14 +55,40 @@
 # In[3]:
 
 
-plate3df_path = pathlib.Path(root_dir / "nf1_painting_repo/3.processing_features/data/single_cell_profiles/Plate_3_sc_feature_selected.parquet").resolve(strict=True)
-plate3pdf_path = pathlib.Path(root_dir / "nf1_painting_repo/3.processing_features/data/single_cell_profiles/Plate_3_prime_sc_feature_selected.parquet").resolve(strict=True)
-plate5df_path = pathlib.Path(root_dir / "nf1_painting_repo/3.processing_features/data/single_cell_profiles/Plate_5_sc_feature_selected.parquet").resolve(strict=True)
+# Set data level
+data_level = "cleaned"
+
+# Main directory path (converted or cleaned data)
+if data_level == "cleaned":
+    data_dir = pathlib.Path(
+        root_dir
+        / "/media/18tbdrive/1.Github_Repositories/nf1_schwann_cell_painting_data/3.processing_features/data/single_cell_profiles/cleaned_sc_profiles"
```

Comment:
consider saving the string in common with line 70 and simply f-stringing the content that is different

Reply:
Added!